Pascal/Pascal.sublime-syntax

```diff
@@ -47,7 +47,7 @@ contexts:
             0: punctuation.definition.comment.pascal
           push:
```

Comment:
one solution here would be to keep your `\n` -> `(?=\n)` changes, and also to change:

    - match: (?!\G)
      pop: true

to

    - match: \n
      scope: comment.line.double-slash.pascal
      pop: true

I'm not sure why Pascal is different from the other syntaxes in that it scopes the whitespace at the beginning of a line, before a comment as `punctuation.whitespace.comment.leading.pascal`, where other syntaxes don't scope it at all. It could be simplified quite a bit if this was removed and thus made consistent with the other syntaxes. What do you think @wbond ?